spotbugs-tests/src/test/java/edu/umd/cs/findbugs/classfile/impl/ClassFactoryTest.java

```diff
@@ -113,12 +114,16 @@ private void assertHasCodeBase(IScannableCodeBase codeBase) throws Exception {
 
     @Test
     public void acceptConstantDynamic() throws Exception {
-        String fileName = "../spotbugsTestCases/src/classSamples/recordCompileWithBazel6.2/Foo.clazz";
+        ArrayList<String> fileNames = new ArrayList<>();
```

Comment:
Could you please add a comment about the class files (how to recreate them), and refer the issue here?